<launch>
	<arg name="name" default="ros_object_detector_client" />
	<arg name="tran_path" default="/home/hjpark/catkin_ws_test/src/ros_object_detector" />
	<arg name="test_image_path_for_object" default="can_third/can_data3/test_images/1.jpg" />
	
	<node pkg="ros_object_detector" type="ros_object_detector_client" respawn="true" name="$(arg name)" >
		<!-- ORANGE -->		
		<param name="test_image_path_for_object" type="string" value="$(arg tran_path)/$(arg test_image_path_for_object)" />
		<!-- internal name -->
		<param name="name" value="$(arg name)" />
	</node>
</launch>
